.travis.yml

```diff
@@ -0,0 +1,6 @@
+language: java
+script: 
+- ./gradlew build
+sudo: required
+jdk:
+  - oraclejdk8
```

Comment:
Reproducing two suggestions in the documentation:
1) [Peculiarities with gradle](https://docs.travis-ci.com/user/languages/java/#Caching)
They recommend adding this:
```
before_cache:
  - rm -f  $HOME/.gradle/caches/modules-2/modules-2.lock
  - rm -fr $HOME/.gradle/caches/*/plugin-resolution/
cache:
  directories:
    - $HOME/.gradle/caches/
    - $HOME/.gradle/wrapper/
```
2) [Java 8 projects](https://docs.travis-ci.com/user/languages/java/#Testing-Against-Multiple-JDKs)
They recommend adding this (they also recommend `sudo : false` in the same section but I guess we have to ignore that and see how it goes with the new additions):
```
addons:
  apt:
    packages:
      - oracle-java8-installer
```